pkg/karmadactl/apply.go

```diff
@@ -44,14 +44,14 @@ var (
 		Alpha Disclaimer: the --prune functionality is not yet complete. Do not use unless you are aware of what the current state is. See https://issues.k8s.io/34274.
 		
 		Note: It implements the function of 'kubectl apply' by default. 
-		If you want to propagate them into member clusters, please use 'kubectl apply --all-clusters'.`)
+		If you want to propagate them into member clusters, please use 'karmadactl apply --all-clusters'.`)
 
 	applyExample = templates.Examples(`
 		# Apply the configuration without propagation into member clusters. It acts as 'kubectl apply'.
 		%[1]s apply -f manifest.yaml
 
 		# Apply resources from a directory and propagate them into all member clusters.
-		%[1]s apply -f dir/ --all-clusters`)
+		%[1]s apply -k dir/ --all-clusters`)
```

Comment:
Why changing to `-k`?

Reply:
The Usage is `-k DIRECTORY`:
<img width="821" alt="image" src="https://user-images.githubusercontent.com/76980726/179353895-aae0088d-8181-430d-b279-0b9f62c07db7.png">

Reply:
```sh
-f, --filename=[]: that contains the configuration to apply
-k, --kustomize='': Process a kustomization directory. This flag can't be used
together with -f or -R.
-R, --recursive=false: Process the directory used in -f, --filename
recursively. Useful when you want to manage related manifests organized within
the same directory.
```

@my-git9  `DIRECTORY ` is also accepted by the `-f` flag. If changing to `-k`, the comment should be changed at same time. i.e.

```sh
  # Apply resources from a directory containing kustomization.yaml - e.g.
dir/kustomization.yaml, and propagate them into all member clusters
 kubectl apply -k dir/ --all-cluster
```

you can add a new example about `-k` instead of changing it.

Reply:
OK, it has been adjusted

Reply:
OK, it has been adjusted